anime_standalone.py

```diff
@@ -0,0 +1,224 @@
+# importing the requests library 
+import requests,os,subprocess
+import sys
+import signal, psutil
+from bs4 import BeautifulSoup
+import re
+from datetime import datetime
+import locale
+import time
+import concurrent.futures
+import wget
+
+#config
+movie_folder = "download/"
+download_path = "download/"
+leng = True #solo anime in italiano (utile per gli anime doppiati, es: SAO)
+all = True #anime correlati
+#
+
+only_link = list()
+list_link = list()
+correlati_list = list()
+anime = {}
+all_ep = {}
+titolo = ""
+season = 0
+season_num = 0
+def kill_child_processes(parent_pid, sig=signal.SIGTERM):
+    try:
+        parent = psutil.Process(parent_pid)
+    except psutil.NoSuchProcess:
+        return
+    children = parent.children(recursive=True)
+    for process in children:
+        process.send_signal(sig)
+
+def checkDownload_Path(crawl_path):
+    if(not os.path.isdir(crawl_path)):
+        os.makedirs(crawl_path)
+
+def find_between( s, first, last ):
+    try:
+        start = s.index( first ) + len( first )
+        end = s.index( last, start )
+        return s[start:end]
+    except ValueError:
+        return ""
+
+def download(url, file_name):
+    print(os.path.join(download_path,file_name.split("_")[0]))
+    checkDownload_Path(os.path.join(download_path,file_name.split("_")[0]))
+    with open(os.path.join(download_path,file_name.split("_")[0],file_name), "wb") as file:
+        response = requests.get(url)
+        file.write(response.content)
+
+def create_crawl():
+    crwd = ""
+    #creo un file vuoto, se presente sovrascrivo
+    checkDownload_Path(download_path) #verifico che path esista
+    print("Rilevati %d episodi"%len(list_link))
+    while True:
+        try:
+            wantedEps = input("Dammi Range Episodi (1:2) ").split(":")
+            start, finish = int(wantedEps[0]), int(wantedEps[1])
+            if(start > 0 and finish > 0):
```

Comment:
```suggestion
            if(start > 0 and finish > 0 and finish <= len(list_link) and start <= len(list_link)):
```
Copre  tutti i  casi